src/_includes/graphql/requisition-list.md

```diff
@@ -0,0 +1,10 @@
+The `RequisitionList` object contains the following attributes.
```

Comment:
Delete this file. I will be merging this PR into the `requisition-list` branch. #8202 (which has been merged), added this file.